projects/ngrx.io/content/examples/store-walkthrough/src/app/tests/app.component.1.spec.ts

```diff
@@ -96,3 +97,93 @@ describe('AppComponent', () => {
     //#enddocregion mockSelector
   });
 });
+
+//#docregion resetMockSelector
+describe('AppComponent reset selectors', () => {
+  let store: MockStore<AppState>;
```

Comment:
Just like `Store`, we want to remove the generic.

```suggestion
  let store: MockStore;
```